### PR TITLE
quickfix: allow to import logs from any document in the project

### PR DESCRIPTION
**Pull request by geclos**
Also fixes the issue where importing logs sometimes did not work

### CODE REVIEW

File: apps/web/src/app/api/documents/[projectId]/for-import/route.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      }: {
+        params: { projectId: string }
+        workspace: Workspace
+      },
+    ) => {
+      const { projectId } = params
+
+      if (!projectId) {
+        throw new BadRequestError('Project ID is required')
+      }
```
Comment (csansoon) [inline apps/web/src/app/api/documents/[projectId]/for-import/route.ts:24]: Can we use zod for input validation in API routes like we do with server actions?

File: packages/core/src/repositories/documentVersionsRepository/index.ts [hunk 2]
```diff
@@ -252,4 +252,16 @@ export class DocumentVersionsRepository extends Repository<
...
+    const documents = await this.db
+      .selectDistinct({
+        documentUuid: this.scope.documentUuid,
+        path: this.scope.path,
+      })
+      .from(this.scope)
+      .where(eq(this.scope.projectId, projectId))
+
+    return Result.ok(documents)
+  }
```
Comment (csansoon) [inline packages/core/src/repositories/documentVersionsRepository/index.ts:266]: What if the document has been deleted? This list could get annoyingly long on long-lived projects

Comment (csansoon) [reply packages/core/src/repositories/documentVersionsRepository/index.ts:266]: A nice problem to have, though

**Review by csansoon — DISMISSED** [review]

File: apps/web/src/app/api/documents/[projectId]/for-import/route.test.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        },
+      },
+    })
+    mockParams = { projectId: String(project.id) }
+
+    mockWorkspace = workspace
+    mockDocuments = documents
+  })
+
+  it('should return 400 if projectId is missing', async () => {
```
Comment (andresgutgon) [inline apps/web/src/app/api/documents/[projectId]/for-import/route.test.ts:38]: What about checking user logged in?

**Review by andresgutgon — DISMISSED** [review]

File: packages/core/src/repositories/documentVersionsRepository/index.ts (hunk 2 above)
Comment (geclos) [reply packages/core/src/repositories/documentVersionsRepository/index.ts:266]: 💯 good point!

File: apps/web/src/app/api/documents/[projectId]/for-import/route.ts (hunk 1 above)
Comment (geclos) [reply apps/web/src/app/api/documents/[projectId]/for-import/route.ts:24]: we would need to create a middleware for that, totally doable

File: apps/web/src/app/api/documents/[projectId]/for-import/route.test.ts (hunk 3 above)
Comment (geclos) [reply apps/web/src/app/api/documents/[projectId]/for-import/route.test.ts]: 💯

**Pull request merged**